### DISCUSSION TITLE
PR feat(route): add fisher-spb news #7583

### DISCUSSION THREAD

**Discussion by denis-ya** [DIYgod/RSSHub#8043] [Q&A] [answered]
Which steps should be done for this PR to be applied?
[#7583](https://github.com/DIYgod/RSSHub/pull/7583)

**Comment by NeverBehave** [comment, marked as answer]
Hi, we are currently implementing our script config standard. It may take a while to pass but we will ask all pending pr to follow it before merge -- current router.js file is too heavy and hard to maintain. 

You might track the progress here: https://github.com/DIYgod/RSSHub/pull/8224
